### PR TITLE
Feat: validate drag and drop component

### PR DESCRIPTION
**Pull request by MNAlfiansyah**
## Changes

- adding validation to drag and drop component 

## Evidence
title: Component Drag And Drop
project: Jabar Super Apps
participants: @rayfajars @johndy2742 @ristianaditya @MNAlfiansyah

### CODE REVIEW

File: dashboard/components/Base/DragAndDropFileMultiple/index.vue [hunk 1]
```diff
@@ -35,7 +35,7 @@
             ref="file"
             type="file"
             class="hidden"
-            :disabled="dataFilesMultiple.length === 0 ? false : true"
+            :disabled="dataFilesMultiple.length === 0 ? false : disabled"
```
Comment (rayfajars) [inline dashboard/components/Base/DragAndDropFileMultiple/index.vue:38]: Why not directly use the value from its props?

File: dashboard/components/Base/DragAndDropFileMultiple/index.vue [hunk 2]
```diff
@@ -87,7 +87,7 @@
 
               <div class="flex items-center">
                 <div class="flex items-center">
-                  <img :src="data.url != '' ? data.url : fileDocument(data)" class="" />
+                  <img :src="data.url !== '' ? data.url : fileDocument(data)" class="" />
```
Comment (rayfajars) [inline dashboard/components/Base/DragAndDropFileMultiple/index.vue:90]: is this class used?

**Review by rayfajars — APPROVED** [review]

**Review by marsellavaleria19 — APPROVED** [review]

File: dashboard/components/Base/DragAndDropFileMultiple/index.vue (hunk 1 above)
Comment (MNAlfiansyah) [reply dashboard/components/Base/DragAndDropFileMultiple/index.vue:38]: the length check is handle for disabled for upload more than 1 photo, its temporary kang until the service ready

File: dashboard/components/Base/DragAndDropFileMultiple/index.vue (hunk 2 above)
Comment (MNAlfiansyah) [reply dashboard/components/Base/DragAndDropFileMultiple/index.vue]: updated kang [bc01e0e](https://github.com/jabardigitalservice/event-management-system/pull/82/commits/bc01e0ec029421b0a16106ea7f28bc59306f8afb)

**Pull request merged**